tests/framework/validators/UniqueValidatorTest.php

```diff
@@ -331,6 +331,31 @@ public function testPrepareParams()
         $result = $this->invokeMethod(new UniqueValidator(), 'prepareConditions', [$targetAttribute, $model, $attribute]);
         $expected = ['val_attr_b' => 'test value b', 'val_attr_c' => 'test value a'];
         $this->assertEquals($expected, $result);
+
+        // Add table prefix for column name.
+        $model = Profile::findOne(1);
+        $attribute = 'id';
+        $targetAttribute = 'id';
+        $result = $this->invokeMethod(new UniqueValidator(), 'prepareConditions', [$targetAttribute, $model, $attribute]);
+        $expected = [Profile::tableName() . '.' . $attribute => $model->id];
+        $this->assertEquals($expected, $result);
+    }
+
+    public function testGetTargetClass()
+    {
+        // Filled property targetClass.
+        $validator = new UniqueValidator();
+        $validator->targetClass = 'target class';
```

Comment:
this is not a real example of a targetClass right ;). Please strive to create a unit test that is instantly understable and also stable for the future. This is likely to break when is check on a real classname.